docker/parabol-ubi/docker-host-st/README.md

```diff
@@ -7,9 +7,11 @@ To run the Parabol UBI in single tenant mode (e.g. simple docker-compose on a do
 3. Update docker-compose.yaml `image: #image:tag` with your built image tag from `step (1.)`
 4. Run `docker-compose up -d` to deploy the local stack. You can run `docker-compose down` to terminate the local stack
 5. Check logs via command `docker logs <name>-app-1` and wait for the following output to appear
+
 ```shell
 🔥🔥🔥 Server ID: 0. Ready for Sockets: Port 3000 🔥🔥🔥
 💧💧💧 Server ID: 0. Ready for GraphQL Execution 💧💧💧
 💧💧💧 Server ID: 01. Ready for GraphQL Execution 💧💧💧
 ```
-6. Run command `docker exec <name>-app-1 yarn postdeploy`
+
+6. Run command `docker exec <name>-app-1 yarn predeploy`
```

Comment:
This isn't correct. Step 6 should be removed and yarn predeploy added to the [docker-compose` file](https://github.com/ParabolInc/parabol/blob/1a3dcd45d0f71bee2a126e0c801506c3f950246e/docker/parabol-ubi/docker-host-st/docker-compose.yaml#L40).

One more note: this single tenant deployment do not work with our current pipeline Docker images. I will create an issue to adapt that, probably after the GCP migration.

Reply:
done! I'm betting there's some other pieces that are broken, which is totally OK! We can fix those build steps later

Reply:
Indeed. I create [this issue](https://github.com/ParabolInc/parabol/issues/8448) to track what we have to do and do it whenever we have the time